### PR TITLE
feat(api): Move permissions and isSuperuser attributes to user details

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

File: src/sentry/api/serializers/models/user.py [hunk 1]
```diff
@@ -143,23 +141,30 @@ class DetailedUserSerializer(UserSerializer):
...
         return attrs
 
     def serialize(self, obj, attrs, user):
         d = super(DetailedUserSerializer, self).serialize(obj, attrs, user)
+        # XXX(dcramer): we dont use is_active_superuser here as we simply
+        # want to tell the UI that we're an authenticated superuser, and
+        # for requests that require an *active* session, they should prompt
+        # on-demand. This ensures things like links to the Sentry admin can
+        # still easily be rendered.
+        d['isSuperuser'] = user.is_superuser
```
Comment (dcramer) [inline src/sentry/api/serializers/models/user.py]: *this* is a change in behavior in that before this you could never identify if a user in Sentry was a superuser unless that user was you. I think its an acceptable risk change.

**Comment by dcramer** [comment]
I need to actually QA this more (also fix build), but wanted to get the review available. I mostly need to confirm the frontend wont have any issues with the removal of permissions from the user_index endpoint. It really shouldn't so I think we'll be safe here.

**Comment by dcramer** [comment]
Did a quick QA pass and seems generally stable.

**Review by billyvg — APPROVED** [review]
lgtm

**Pull request merged**